### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by fullofcaffeine** [HaxeFoundation/HX101_Introduction-to-Cross-platform-Development-with-Haxe#2]
Will this ever be released?

**Comment by fullofcaffeine** [comment]
Ping, I think this repo is just taking up space and confusing people. Maybe it should be removed?

**Comment by andyli** [comment]
Yeah, I guess so. Hope I will actually work on it soon. I'm going to remove this repo in the mean time.
